lib/rugged_adapter/git_layer_rugged.rb

```diff
@@ -126,6 +126,10 @@ def stats
         @stats ||= build_stats
       end
 
+      def parent
+        Gollum::Git::Commit.new(@commit.parents.first)
```

Comment:
To take care of the 'no parent' edge case, I assume it should be something like this:

`@commit.parents.empty? ? nil : Gollum::Git::Commit.new(@commit.parents.first)`

Reply:
Thanks! Fixed accordingly.